market-source.yaml

```diff
@@ -1,13 +1,10 @@
 # camel-k: language=yaml
-# camel-k: dependency=camel-jackson
 
 - from:
     uri: "timer:update"
     parameters:
       period: 10000
     steps:
-    - to: "xchange:binance?service=marketdata&method=ticker&currencyPair=BTC/USDT"
-    - marshal:
-        json: {}
+    - to: "https://www.binance.com/api/v3/ticker/price?symbol=BTCUSDT"
```

Comment:
I think the description in readme must be updated as the xchange component is no more used https://github.com/openshift-integration/camel-k-example-knative/blame/d55f788f7d03501f099f0b439b80e632317581de/readme.didact.md#L10 (unless I missed a usage somewhere else?)

Reply:
@apupier That's right thanks! My plan though is to use the Camel XChange component again once the binance dependency update that you have fixed the other day can be used in the example.